doc/source/user_guide/timeseries.rst

```diff
@@ -1796,6 +1776,34 @@ level of ``MultiIndex``, its name or location can be passed to the
 
    df.resample('M', level='d').sum()
 
+.. _timeseries.iterating-label:
```

Comment:
why did you move this? (not averse, just want to know the reason)

Reply:
It looked out of place where it was. I put with the rest of the resampling related